apps/pgadmin4/8.13/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   pgadmin4:
-    image: dpage/pgadmin4:8.13
+    image: dpage/pgadmin4:8.14
     container_name: ${CONTAINER_NAME}
     restart: always
     labels:
```

Comment:
The only difference between lines -1 and +1 is the image version from `8.13` to `8.14`. This change alone does not introduce any immediate irregularities, but it's good practice to ensure compatibility with your application requirements. If there are no updates required beyond this version, you might want to consider updating your deployment scripts or configuration files to use the newly available image version for consistency.